Automattic/newspack-plugin
# refactor(ia): advertising display ads

### All Submissions:


### Changes proposed in this Pull Request:

Refactors Advertising slugs. The initial intention was to structure slugs as a representation of location i.e. `{top-level-admin-page-name}-{submenu-name}`. 
* Newspack / Dashboard - `newspack-dashboard`
* Newspack / Settings - `newspack-settings`
* Advertising / Display Ads - `advertising-display-ads`

As the project progressed a similar pattern was followed with the addition of a `newspack-` prefix.
This PR introduces `newspack-` prefix to Advertising. 

### How to test the changes in this Pull Request:

1. Checkout branch and build assets - `git checkout origin/feat/ia-consistent-slugs && npm run build`
2. Navigate to _/wp-admin/admin.php?page=newspack-advertising-display-ads#/_
3. Ensure Advertising / Display Ads loads and functions as expected.
4. Navigate to _/wp-admin/admin.php?page=newspack-listings-settings_
5. Ensure Listings / Settings loads and functions as expected.

### Other information:
